sql/sql_locale.cc

```diff
@@ -893,6 +893,42 @@ MY_LOCALE my_locale_hi_IN
 );
 /***** LOCALE END hi_IN *****/
 
+/***** LOCALE BEGIN ka_GE: Georgian - Georgia *****/
+static const char *my_locale_month_names_ka_GE[13] = 
+ {"იანვარი","თებერვალი","მარტი","აპრილი","მაისი","ივნისი","ივლისი","სექტემბერი","ოქტომბერი","ნოემბერი","დეკემბერი", NullS };
+static const char *my_locale_ab_month_names_ka_GE[13] = 
+ {"იან","თებ","მარ","აპრ","მაი","ივნ","ივლ","აგვ","სექტ","ოქტ","ნოე","დეკ", NullS };
+static const char *my_locale_day_names_ka_GE[8] = 
+ {"ორშაბათი","სამშაბათი","ოთხშაბათი","ხუთშაბათი","პარასკევი","შაბათი","კვირა", NullS };
+static const char *my_locale_ab_day_names_ka_GE[8] = 
+ {"ორშ","სამშ","ოთხშ","ხუთშ","პარ","შაბ","კვ", NullS };
+static TYPELIB my_locale_typelib_month_names_ka_GE = 
+ { array_elements(my_locale_month_names_ka_GE)-1, "", my_locale_month_names_ka_GE, NULL };
+static TYPELIB my_locale_typelib_ab_month_names_ka_GE = 
+ { array_elements(my_locale_ab_month_names_ka_GE)-1, "", my_locale_ab_month_names_ka_GE, NULL };
+static TYPELIB my_locale_typelib_day_names_ka_GE = 
+ { array_elements(my_locale_day_names_ka_GE-1, "", my_locale_day_names_ka_GE, NULL };
```

Comment:
Missed off the braces around `my_locale_day_names_ka_GE` on this line, which is causing the compile failure.